models/comment.go

```diff
@@ -0,0 +1,25 @@
+package models
+
+import (
+	"fmt"
+	"time"
+
+	validation "github.com/go-ozzo/ozzo-validation"
+)
+
+type Comment struct {
+	ID        string
+	Content   string
+	CreatedBy string
+	CreatedAt time.Time
+	PostID    string
+}
+
+func (c *Comment) Validate() string {
+	err := validation.ValidateStruct(c, validation.Field(&c.Content, validation.Required))
+	if err != nil {
+		return fmt.Sprint(err)
```

Comment:
this is not good
please read these articles: 
- https://habr.com/ru/company/otus/blog/453806/
- https://habr.com/ru/company/mailru/blog/473658/